tests/all/main.rs

```diff
@@ -264,28 +277,30 @@ impl Project {
     }
 
     fn generate_js_entry(&mut self, modules: Vec<PathBuf>) {
-        let mut runjs = r#"
-            import * as process from "process";
-        "#.to_string();
+        let mut runjs = if self.headless {
+            "".to_string()
+        } else {
+            "import * as process from \"process\";".to_string()
```

Comment:
`r#` was more expressive for me 😄 

Is there any advantage using this over raw string ?

Reply:
My highlighting was starting to freak out and I just did that as a temporary fix. Agreed that `r#` strings are better here.